index.js

```diff
@@ -36,38 +39,68 @@ app.post('/login', csrf, async (req, res, next) => {
 // logged in only zone
 app.use(verifyEmail);
 
+// get the student who has just logged in
 app.get('/users/:uid', (req, res) => {
     // TODO: get user information?
     res.status(200);
     res.send(req.body.user);
 });
 
-// admin only zone
-app.use(verifyAdmin);
-
+// add a student with uid to the reception list
 app.post('/users/:uid/recept', async (req, res) => {
     const uid = parseInt(req.params.uid);
     if (isNaN(uid)) {
         res.status(400);
         res.send('uid must be a number');
         return;
+    } else if(client.student.findUnique({where: {id: uid}}) == null) {
+        res.status(400);
+        res.send('uid not found');
+        return;
     }
```

Comment:
Cái này bà đổi thành `uid == req.body.user.id` luôn giúp tui nha, nó vừa check được coi có tồn tại sinh viên không, vừa check coi đúng người check-in không á (không có vụ check-in chéo được)

```suggestion
    } else if (uid == req.body.user.id) {
        res.status(400);
        res.send('uid must be yours');
        return;
    }
```